conf/repository.spring.properties

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.

Reply:
Removed in git and we will pass this in k8s